pkgs/development/compilers/rust/print-hashes.sh

```diff
@@ -19,6 +19,7 @@ PLATFORMS=(
   aarch64-apple-darwin
   powerpc64le-unknown-linux-gnu
   riscv64gc-unknown-linux-gnu
+  mips64el-unknown-linux-gnuabi64
```

Comment:
@a-m-joseph Please rebase this PR so it only adding this line.

Reply:
Done.